index_websocket.js

```diff
@@ -0,0 +1,97 @@
+const http = require("http");
+const open = require("open");
+const fs = require("fs");
+const WebSocketServer = require("websocket").server; // websocket server class
+
+let webSocket = null;
+// connection variable, to refer to the web socket connection to the client once connection is established
+let connection = null;
+// default port used if none is specified by the user
+let defaultPort = 9101;
```

Comment:
Let's use `CAPITALIZED_WITH_UNDERSCORES` to name constants and use `const` wherever possible.
```suggestion
const DEFAULT_PORT = 9101;
```